packages/plugin-msdynamic-api/src/utils.ts

```
@@ -215,14 +219,322 @@ export const consumeCategory = async (
+      Responsibility_Center: config.responsibilityCenter || '',
+      Sync_Type: config.syncType || '',
+      Mobile_Phone_No: customer?.primaryPhone || '',
+      VAT_Bus_Posting_Group: config.vatBusPostingGroup || '',
+      Payment_Terms_Code: config.paymentTermsCode || '',
+      Payment_Method_Code: config.paymentMethodCode || 'CASH',
+      Customer_Price_Group: config.customerPricingGroup || '',
+      Prices_Including_VAT: true,
+      BillType: config.billType || 'Receipt',
+      Location_Code: config.locationCode || '',
+      Deal_Type_Code: config.dealType || 'NORMAL',
+      Salesperson_Code: user && user.employeeId,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)